src/promptflow/pf.autocomplete.sh

```diff
@@ -0,0 +1,11 @@
+case $SHELL in
```

Comment:
We should add a changelog for this feature.

Reply:
Added in src/promptflow/CHANGELOG.md in my last commit